scipy/stats/tests/test_distributions.py

```diff
@@ -717,6 +717,32 @@ def test_norm(self):
         pdf2 = stats.norm.pdf(points, scale=2**-.5)
         assert_almost_equal(pdf1, pdf2)
 
+    def test_rvs(self):
+        np.random.seed(0)
+        # 0 < beta < 1
+        dist = stats.gennorm(0.5)
+        rvs = dist.rvs(size=1000)
+        assert stats.kstest(rvs, dist.cdf).pvalue > 0.1
+        # beta = 1
+        dist = stats.gennorm(1)
+        rvs = dist.rvs(size=1000)
+        rvs_laplace = stats.laplace.rvs(size=1000)
+        assert stats.ks_2samp(rvs, rvs_laplace).pvalue > 0.1
+        # beta = 2
+        dist = stats.gennorm(2)
+        rvs = dist.rvs(size=1000)
+        rvs_norm = stats.norm.rvs(size=1000)
+        assert stats.ks_2samp(rvs * (2)**(1/2), rvs_norm).pvalue > 0.1
```

Comment:
```suggestion
        rvs_norm = stats.norm.rvs(scale=1/2**0.5, size=1000)
        assert stats.ks_2samp(rvs, rvs_norm).pvalue > 0.1
```
Minor point, but just to make it more obvious for me that it matches what we say in the `gennorm` documentation:

![image](https://user-images.githubusercontent.com/6570539/149586079-684cfd5c-73fe-4537-8bd9-83b85febc44d.png)